### DISCUSSION TITLE
Where to get info about about branch names for each QuML player version ?

### DISCUSSION THREAD

**Discussion by devendra-tarento** [Sunbird-inQuiry/Community#72] [Q&A] [answered]
We are using "@project-sunbird/sunbird-quml-player-v9": "5.1.5" in sunbirdEd setup. Now i want to do some enhancement in this library. To do the enhancement we need to clone the relevant branch from the player repo. -> https://github.com/project-sunbird/sunbird-quml-player

Question: I am unable to figure out which branch is containing version "5.1.5" code ?

Please let us know if thee is already a doc which we can refer to know branch <---> version mapping.

cc: @pallakartheekreddy @Krishnaj20

**Comment by rajnishdargan** [comment, marked as answer]
Hi @devendra-tarento 
You can refer the release notes to find out the branch and version of each release : [release-notes](https://inquiry.sunbird.org/use/release-notes).
@project-sunbird/sunbird-quml-player-v9 is deprecated in [release-5.1.0](https://inquiry.sunbird.org/use/release-notes/inquiry-release-v5.1.0)
The new npm package is [@project-sunbird/sunbird-quml-player](https://www.npmjs.com/package/@project-sunbird/sunbird-quml-player)
Github repo for the QuML player is also changed in release-5.5.0. Please refer the deprecation notes:
https://inquiry.sunbird.org/use/deprecations/release-5.5.0
For any new contribution please use new github repo: https://github.com/Sunbird-inQuiry/player
cc: @pallakartheekreddy @vaibhavbhuva @PoojaJManjooran
